### PR TITLE
fix typo of configure-volume-storage

### PR DESCRIPTION
**Pull request by jianglingxia**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox
maybe --watch move the last better

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5404)

### CODE REVIEW

**Comment by tengqm** [comment]
looks like we are having problems with netlify ...

File: docs/tasks/configure-pod-container/configure-volume-storage.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ restarts. Here is the configuration file for the Pod:
 1. Verify that the Pod's Container is running, and then watch for changes to
 the Pod:
 
-       kubectl get --watch pod redis
+       kubectl get pod redis --watch
```
Comment (zhangxiaoyu-zidif) [inline docs/tasks/configure-pod-container/configure-volume-storage.md]: is there any difference on usage?

Comment (jianglingxia) [reply docs/tasks/configure-pod-container/configure-volume-storage.md]: function is equal,but maybe later looks beauty I think

**Comment by zacharysarah** [comment]
Netlify is passing now, so: merging!

**Pull request merged**